### ISSUE TITLE
Best way to hid the webView

### ISSUE THREAD

**Issue by galaxybeing** [myflashlab/webView-ANE#39]
What is the best way to hide and show the webView?  I am drawing it off screen, but when I re-draw it on screen, it scrolls to the top of the page.  It would be nice if the page position didn't change.  I think if I save the scroll position and the set it when re-drawing on screen I'll see it jump up and down?

**Comment by myflashlab** [comment]
Never underestimate CSS and JavaScript :) maybe you can freeze scrolling inside JS by overrolling CSS. try something like below and it might help. if not, saving scroll position may also be a good solution.

```js
window.preventScroll = function($allow) {
  var body = document.getElementsByTagName('body')[0];

  if ($allow) body.style.overflow = 'hidden';
  else body.style.overflow = 'auto';
};

```

**Comment by galaxybeing** [comment]
It would be great to be able to change the x and y of the webView window without changing it's widtht and height.  Is it that much of an issue to hide and show the webView without it resetting everything?  I'm trying saving and restoring the scroll but it doesn't seem to be working.  It want's to reset to 0, which makes for a sucky user experience.  Imagine if everytime you went to check your email and came back to your web pages and they were all back to 0.  Are the pages actually getting reloaded?

**Comment by myflashlab** [comment]
with the upcoming release of V6.0.0 you will be able to hide/show the webview with a command like ```_ex.hide(true);```

**Comment by galaxybeing** [comment]
It would br great if "hide" could somehow release the keyboard for Android Flash input fields.  (A different bug)



On Sat, Feb 13, 2016 at 3:34 AM -0800, "myflashlab" <notifications@github.com> wrote:












Closed #39.

**Comment by myflashlab** [comment]
Please open a new issue with this so we can add it to the board.
